$NetBSD: patch-configure.ac,v 1.13 2019/09/18 14:12:20 ryoon Exp $

* Don't assume sys/capability.h is guaranteed present on Linux.
* Fix NetBSD detection for libatomics_ops check.
* Fix unportable test(1) construct.
* Adapt /var-paths for pkgsrc defaults.
* Install default config files in example directory; pkgsrc config
  file framework will install them in the right place.
* Avoid hardcoded SDK paths on Darwin.
* Check for memfd_create instead of assuming libc does not define it.
  (Will be included in next release after 11.1)
* On Darwin, do not use macosx-version-min.

--- configure.ac.orig	2018-06-18 13:33:47.000000000 +0000
+++ configure.ac
@@ -21,7 +21,7 @@
 
 AC_PREREQ(2.63)
 
-AC_INIT([pulseaudio],[m4_esyscmd(./git-version-gen .tarball-version)],[pulseaudio-discuss (at) lists (dot) freedesktop (dot) org],[pulseaudio],[http://pulseaudio.org/])
+AC_INIT([pulseaudio],[13.0],[pulseaudio-discuss (at) lists (dot) freedesktop (dot) org],[pulseaudio],[http://pulseaudio.org/])
 AC_CONFIG_SRCDIR([src/daemon/main.c])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_HEADERS([config.h])
@@ -130,7 +130,6 @@ AC_MSG_CHECKING([host operating system])
 case "$host_os" in
     linux*)
         AC_MSG_RESULT([linux])
-        host_has_caps=1
         os_is_linux=1
     ;;
     freebsd*)
@@ -613,6 +612,9 @@ AS_IF([test "x$enable_memfd" = "xyes" &&
 AS_IF([test "x$HAVE_MEMFD" = "x1"],
     AC_CHECK_FUNCS([memfd_create]))
 
+AS_IF([test "x$HAVE_MEMFD" = "x1"],
+    AC_CHECK_FUNCS([memfd_create]))
+
 AC_SUBST(HAVE_MEMFD)
 AM_CONDITIONAL([HAVE_MEMFD], [test "x$HAVE_MEMFD" = x1])
 AS_IF([test "x$HAVE_MEMFD" = "x1"], AC_DEFINE([HAVE_MEMFD], 1, [Have memfd shared memory.]))
@@ -1065,7 +1067,7 @@ AS_IF([test "x$HAVE_DBUS" = "x1"], AC_DE
 
 PA_MACHINE_ID="${sysconfdir}/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID, PA_MACHINE_ID, [D-Bus machine-id file])
-PA_MACHINE_ID_FALLBACK="${localstatedir}/lib/dbus/machine-id"
+PA_MACHINE_ID_FALLBACK="${localstatedir}/db/dbus/machine-id"
 AX_DEFINE_DIR(PA_MACHINE_ID_FALLBACK, PA_MACHINE_ID_FALLBACK,
 	      [Fallback machine-id file])
 
@@ -1364,9 +1366,9 @@ fi
 
 PA_SYSTEM_RUNTIME_PATH="${localstatedir}/run/pulse"
 AX_DEFINE_DIR(PA_SYSTEM_RUNTIME_PATH, PA_SYSTEM_RUNTIME_PATH, [System runtime dir])
-PA_SYSTEM_CONFIG_PATH="${localstatedir}/lib/pulse"
+PA_SYSTEM_CONFIG_PATH="${localstatedir}/db/pulse"
 AX_DEFINE_DIR(PA_SYSTEM_CONFIG_PATH, PA_SYSTEM_CONFIG_PATH, [System config dir])
-PA_SYSTEM_STATE_PATH="${localstatedir}/lib/pulse"
+PA_SYSTEM_STATE_PATH="${localstatedir}/db/pulse"
 AX_DEFINE_DIR(PA_SYSTEM_STATE_PATH, PA_SYSTEM_STATE_PATH, [System state dir])
 
 PA_BINARY=${bindir}/pulseaudio${EXEEXT}
@@ -1378,8 +1380,9 @@ AX_DEFINE_DIR(PACTL_BINARY, PACTL_BINARY
 AC_SUBST(PA_SOEXT, [.so])
 AC_DEFINE(PA_SOEXT, [".so"], [Shared object extension])
 
-AC_SUBST(pulseconfdir, ["${sysconfdir}/pulse"])
-AX_DEFINE_DIR(PA_DEFAULT_CONFIG_DIR, pulseconfdir, [Location of configuration files])
+AC_SUBST(pulseconfdirreal, ["${sysconfdir}/pulse"])
+AC_SUBST(pulseconfdir, ["${EGDIR}"])
+AX_DEFINE_DIR(PA_DEFAULT_CONFIG_DIR, pulseconfdirreal, [Location of configuration files])
 
 #### Mac OSX specific stuff #####
 
@@ -1396,8 +1399,6 @@ AC_ARG_WITH(mac-sysroot,
     mac_sysroot=$withval)
 
 if test "x$os_is_darwin" = "x1" ; then
-    LDFLAGS="$LDFLAGS -mmacosx-version-min=$mac_version_min"
-    CFLAGS="$CFLAGS -mmacosx-version-min=$mac_version_min"
 
     if test "x$mac_sysroot" != "x" ; then
         LDFLAGS="$LDFLAGS -isysroot $mac_sysroot"
